common/txmgr/tracker.go

```diff
@@ -104,7 +104,7 @@ func (tr *Tracker[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) startIntern
 
 	tr.ctx, tr.ctxCancel = context.WithCancel(context.Background())
 
-	if err := tr.setEnabledAddresses(); err != nil {
+	if err := tr.setEnabledAddresses(tr.ctx); err != nil {
```

Comment:
Oh this method was already confused. It is invoked directly from Start(), and spawns a goroutine for background work at the very end, so it should be accepting a `context.Context` from Start() to use for this call and trackAbandonedTxes.

Reply:
Believe I handled this is in the latest revision 👍